src/extension.ts

```diff
@@ -29,41 +28,41 @@ function getSysroot(env: Object): string | Error {
     const rustcSysroot = child_process.spawnSync(
         'rustup', ['run', 'nightly', 'rustc', '--print', 'sysroot'], {env}
     );
-
+    
```

Comment:
Could you avoid inserting all this extra whitespace please

Reply:
Sure; a lot of this was over zealously letting VSCode decide on formatting.